ci/scripts/export-envvars-to-circle-ci.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+set -euo pipefail
+readonly CIRCLE_API_BASE=https://circleci.com/api/v1.1/project/github/spring-cloud-incubator/spring-cloud-app-broker
+
+export_variables_to_circle() {
+	source "../.envrc"
+	#DIRECTOR_SSLCA
+	delete_envvar_from_circle DIRECTOR_SSLCA
+	if [[ -z "$DIRECTOR_SSLCA" ]]; then
+		echo "variable DIRECTOR_SSLCA must be set, make sure .envrc file is up to date"
+		exit 1
+	fi
+	create_envvar_in_circle DIRECTOR_SSLCA "${DIRECTOR_SSLCA}"
+
+	#JUMPBOX_PRIVATE_KEY
+	delete_envvar_from_circle JUMPBOX_PRIVATE_KEY
+	if [[ -z "$JUMPBOX_PRIVATE_KEY" ]]; then
```

Comment:
I wonder if we should be just checking for every var do be there first and then set them afterward so that we don't have some set and some not.

Reply:
Agreed